INSTALL.md

```diff
@@ -38,49 +40,38 @@ Runtime DBus dependencies
  - org.gnome.SessionManager
  - org.gnome.SessionManager.EndSessionDialog
 
-Building the code
------------------
-The simplest case is:
- $ cd ayatana-indicator-session-X.Y.Z
- $ mkdir build
- $ cd build
- $ cmake ..
- $ make
+## For end-users and packagers
 
-Running the tests
------------------
- $ cd ayatana-indicator-session-X.Y.Z
- $ mkdir build
- $ cd build
- $ cmake ..
- $ make
- $ make test
- $ make cppcheck
-
-Generating Test Coverage Reports
---------------------------------
- $ cd ayatan-indicator-session-X.Y.Z
- $ mkdir build-coverage
- $ cd build-coverage
- $ cmake -DCMAKE_BUILD_TYPE=coverage ..
- $ make
- $ make coverage-html
-
-Installation
-------------
+The simplest case is:
 
-what gets installed
-LC_ALL=C /usr/bin/intltool-merge  -x -u --no-translations org.ayatana.indicator.session.gschema.xml.in org.ayatana.indicator.session.gschema.xml
+```
+cd ayatana-indicator-session-X.Y.Z
+mkdir build
+cd build
+cmake ..
+make
+sudo make install
 
+**The install prefix defaults to `/usr`, change it with `-DCMAKE_INSTALL_PREFIX=/some/path`**
 
+## For testers - unit tests only
 
+```
+cd ayatana-indicator-session-X.Y.Z
+mkdir build
+cd build
+cmake -DENABLE_TESTS=ON ..
+make
+make test
+make cppcheck
+```
 
-FIXME: not tested
-To get files that form part of an installation, run a "make install"
-in the build directory. By default, this installs them in the "install"
-subdirectory of the build directory. If you want to install into a
-different directory, use
+## For testers - both unit tests and code coverage
 
-$ cmake -DCMAKE_INSTALL_PREFIX=/usr/local # Or wherever...
-$ make release
-$ make install
+```
+cd ayatan-indicator-session-X.Y.Z
+mkdir build-coverage
```

Comment:
Could we use **build** as a build directory consistently throughout all doc?

Reply:
@tari01: I have made the observation that when re-using a build/ folder for coverage reports, that I have to remove everything from that folder (well, mainly CMakeCache.txt, but well...). The coverage report failed from time to time when trying to reuse an already existing build/ folder.

Reply:
If we re-use the build/ folder, we should remove it and recreate it for coverage reports. The reason of the report failure is unknown to me at the moment.

Reply:
I'm OK with whatever we call it, I just thought since we are doing a fresh `mkdir`, we might as well use **build** to keep it all nice and consistent. Change it if you agree, otherwise just resolve this as-is.